### ISSUE TITLE
Add manual installation instructions

### ISSUE THREAD

**Issue by MarkTiedemann** [denoland/deno_install#130]
As witnessed in https://github.com/denoland/deno/issues/5727 and https://github.com/denoland/deno_install/issues/129, people may not expect Deno to just be an `.exe` in a `.zip`.

I think adding manual installation instructions might help.

To reduce installation complexity, it might also help to provide both a `.zip` and an `.exe` for download. (cc @ry - Is this something you'd consider?)

On a related note, if you download the `.exe` and double-click it on Windows, you'll get a warning that "Windows protected your PC", you have to click "More info", and then "Run anyway" to finally enter the REPL. Fixing this, I think, is something that would help make running Deno for the first time more beginner friendly.

**Comment by divy-work** [comment]
@MarkTiedemann I don't know why would anyone be so lazy to download the `.zip` and extract it?

**Comment by MarkTiedemann** [comment]
@divy-work How is downloading the `.zip` lazy? That's manual work, whereas if you use the installer, it does the work for you automatically. So I think it's the other way around; the installer script is the "lazy" way to install Deno.

I think a lot of people are not familiar with PowerShell... I think "download a zip, extract the zip, double-click the exe" maybe more familiar.

**Comment by divy-work** [comment]
@MarkTiedemann Agreed. I'm of no authority. It's better to have something that helps but I still feel this is not needed.

**Comment by Yamboy1** [comment]
I mean, I don't see any reason why we shouldn't have an exe installer. Just cos there's a command line method doesn't mean it's the best on any platform, heck, there's a reason that i'm going to the trouble of maintaining this: https://aur.archlinux.org/packages/deno-bin, because AUR packages are more typical in an arch / manjaro system than using shell script installers. Also see my comment here: https://github.com/denoland/deno/issues/5727#issuecomment-634394910

**Comment by MarkTiedemann** [comment]
> I still feel this is not needed.

I think the two linked issues above show that this may be helpful for beginners. Most Windows users have extracted a zip-file and double-clicked an exe. Running Powershell commands, I feel like, is not a very common thing.

> AUR packages are more typical in an arch / manjaro system than using shell script installers

There are also multiple package managers for Windows, for example, Scoop, Chocolatey and NuGet, and we already support both Scoop and Choco.

> I don't see any reason why we shouldn't have an exe installer.

Me neither. In fact, I have started working on a private C++ project I call "deno-up". Manual install instructions and a `setup.exe` are not mutually exclusive.
